packages/documentation/src/content/docs/integration/getting-started.md

```diff
@@ -230,11 +230,11 @@ Example Successful Response
 ```
 {
   "data": {
-    "createPaymentPointer": {
+    "createWalletAddress": {
```

Comment:
Will this be a new mutation or removing the old one?

Reply:
This should be a new mutation, in https://github.com/interledger/rafiki/tree/nl-rename-all-payment-pointer the `createPaymentPointer` mutation will be refactored to `createWalletAddress`.